test-manifest.yaml

```diff
@@ -18,6 +18,14 @@ env:
   VAR_A: Greetings
   VAR_B: World
 
+triggers:
+- type: pipeline
+  ref: github.com/estafette/estafette-ci-manifest/build
+  branch: release
```

Comment:
It still feels a bit verbose, but the filter/branch can be dropped if you want to trigger on any upstream. Perhaps the filtering needs to be split in _remote_ (the source of the trigger) and _local_ (what branch to run locally if triggered).